kolibri/plugins/user/assets/src/views/SignInPage.vue

```diff
@@ -2,192 +2,100 @@
-                  :text="coreString('signInLabel')"
-                  :primary="true"
-                  :disabled="busy"
-                />
-              </div>
-            </form>
-
-            <p class="create">
-              <KRouterLink
-                v-if="canSignUp"
-                :text="$tr('createAccountAction')"
-                :to="signUpPage"
-                :primary="true"
-                appearance="flat-button"
-              />
-            </p>
-            <div slot="options">
-              <component :is="component" v-for="component in loginOptions" :key="component.name" />
-            </div>
-            <p
-              v-if="showGuestAccess"
-              class="guest small-text"
+    <AuthBase>
+      <div
+        v-if="hasMultipleFacilities && selectedFacility.name"
+        style="margin-bottom: 8px;"
+      >
+        <span style="margin-right: 8px;">
+          {{ $tr("signInToFacilityLabel", { facility: selectedFacility.name }) }}
+        </span>
+        <KRouterLink
+          :text="$tr('changeLabel')"
+          :to="{
+            name: PageNames.FACILITY_SELECT,
+            query: { next: PageNames.SIGN_IN, backTo: PageNames.SIGN_IN }
+          }"
+        />
+      </div>
+      <form ref="form" class="login-form" @submit.prevent="signIn">
+        <UiAlert
+          v-if="invalidCredentials"
+          type="error"
+          :dismissible="false"
+        >
+          {{ $tr('signInError') }}
+        </UiAlert>
+        <transition name="textbox">
+          <KTextbox
+            id="username"
+            ref="username"
+            v-model="username"
+            autocomplete="username"
+            :autofocus="!hasMultipleFacilities"
+            :label="coreString('usernameLabel')"
+            :invalid="usernameIsInvalid"
+            :invalidText="usernameIsInvalidText"
+            @blur="handleUsernameBlur"
+            @input="showDropdown = true"
+            @keydown="handleKeyboardNav"
+          />
+        </transition>
+        <transition name="list">
+          <div class="suggestions-wrapper">
+            <ul
+              v-if="simpleSignIn && suggestions.length"
+              v-show="showDropdown"
+              class="suggestions"
+              :style="{ backgroundColor: $themeTokens.surface }"
             >
-              <KExternalLink
-                :text="$tr('accessAsGuest')"
-                :href="guestURL"
-                :primary="true"
-                appearance="basic-link"
+              <UiAutocompleteSuggestion
+                v-for="(suggestion, i) in suggestions"
+                :key="i"
+                :suggestion="suggestion"
+                :style="suggestionStyle(i)"
+                @mousedown.native="fillUsername(suggestion)"
               />
-            </p>
+            </ul>
           </div>
+        </transition>
+        <transition name="textbox">
+          <KTextbox
+            v-if="needPasswordField"
```

Comment:
The `learner_can_login_with_no_password` doesn't seem to be reflected when I switch between facilities with differing settings. Maybe the facility settings need to be reloaded when you switch?

Reply:
This was a great catch thanks JB!